server/utils/http.ts

```diff
@@ -0,0 +1,17 @@
+import { request } from 'undici';
+
+interface IPostParams {
+	url: string;
+	body: any;
+	headers: any;
```

Comment:
Cant this be `string[]` instead of any? (looks like undici's types support headers as `IncomingHTTPHeaders | string[] | null` (https://github.com/nodejs/undici/blob/9a7d068cb2b8b29c2457bb6caf699295debcee44/types/dispatcher.d.ts#L40)